be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -797,7 +797,8 @@ void SegmentIterator::_init_current_block(
         auto cid = _schema.column_id(i);
         auto column_desc = _schema.column(cid);
 
-        if (_is_pred_column[cid]) { //todo(wb) maybe we can release it after output block
+        if (_is_pred_column[cid] ||
```

Comment:
Please add some comment here, explain why i will GT block->columns()

Reply:
have add comment in init `_current_return_columns`